### PR TITLE
log-in page html

### PR DESCRIPTION
**Pull request by bayan-404**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "description": "Yakoob-Karmel-Bayan-Kholoud",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (hshahwan) [inline package.json:7]: could you update your script?

File: Public/index.html [hunk 2]
```diff
@@ -28,11 +28,15 @@
         <section class="">
             <div class="container">
                 <div class="row" id="">
-
+                  <h1>log in </h1>
+                   <form id="logInForm" class ="logInForm">
```
Comment (hshahwan) [inline Public/index.html]: what the purpose of adding id and class for all elements?

File: package.json (hunk 1 above)
Comment (bayan-404) [reply package.json:7]: this is just a mistake ,we fixed it

File: Public/index.html (hunk 2 above)
Comment (bayan-404) [reply Public/index.html]: because I'll use them later

**Pull request merged**